apps/web/modules/ee/contacts/api/client/[environmentId]/user/lib/update-user.ts

```diff
@@ -0,0 +1,135 @@
+import { contactCache } from "@/lib/cache/contact";
+import { updateAttributes } from "@/modules/ee/contacts/lib/attributes";
+import { prisma } from "@formbricks/database";
+import { getEnvironment } from "@formbricks/lib/environment/service";
+import { getOrganizationByEnvironmentId } from "@formbricks/lib/organization/service";
+import { ResourceNotFoundError } from "@formbricks/types/errors";
+import { getContactByUserIdWithAttributes } from "./contact";
+import { getUserState } from "./user-state";
+
+export const updateUser = async (
+  environmentId: string,
+  userId: string,
+  device: "phone" | "desktop",
+  attributes?: Record<string, string>
+) => {
+  const environment = await getEnvironment(environmentId);
+
+  if (!environment) {
+    throw new ResourceNotFoundError(`environment`, environmentId);
+  }
+
+  const organization = await getOrganizationByEnvironmentId(environmentId);
```

Comment:
Do we need to get the organization here? It seems like the organization is not being used somewhere except for the error.
Please remove if unused.